homeassistant/components/switchbot_cloud/sensor.py

```diff
@@ -0,0 +1,95 @@
+"""Platform for sensor integration."""
+
+from __future__ import annotations
```

Comment:
I am not sure if __future__ is used or not

Reply:
Removed